serverless.yml

```diff
@@ -207,7 +207,7 @@ functions:
     handler: sns/user/lifecycle/create.handler
     description: Handle all `user.lifecycle.create` okta events
     memorySize: 256
-    timeout: 6
+    timeout: 30
```

Comment:
is the request timeout on the gr call less than 30 seconds?

Reply:
GR database timeout is 30 seconds, but if this takes anywhere near 30 seconds, we've got an issue. Once the lambda runs for a while, we'll drop this down to slightly above the max execution time.